### PR TITLE
Bug fixes for header

### PR DESCRIPTION
**Pull request by Ellmen**
The scroll up animation still isn't working but everything else is.

Check out the team page on the wiki!

### CODE REVIEW

**Comment by alexanian** [comment]
The Z-index is definitely fixed, I'm happy to merge

**Comment by Ellmen** [comment]
Wait! I'll fix the dropdown thing

File: wiki/CSS/style.css [hunk 1]
```diff
@@ -151,20 +151,25 @@ header {
...
   text-align: center;
-  margin-top: 16px;
 }
 
 /* Styling for upper nav */
 
 nav {
+  margin-top: 16px !important;
   transition: all 1s !important;
   -webkit-transition: top 0.5s; /* Safari */
```
Comment (jrhawley) [inline wiki/CSS/style.css]: can you include the other versions of transition? like -o-transition and -moz-transition?

Comment (Ellmen) [inline wiki/CSS/style.css]: sure, support all the browsers!

**Comment by jrhawley** [comment]
Cool. Looks like the branch has conflicts you'll have to resolve before merging

**Comment by Ellmen** [comment]
Should be good now

**Pull request merged**